pkg/cli/rm_helper.go

```diff
@@ -123,7 +125,11 @@ func deleteOthers(ctx context.Context, c client.Client, arg string) (bool, error
 }
 
 func removeAcorn(ctx context.Context, c client.Client, arg string, ignoreCleanup, wait bool) error {
-	app, err := c.AppDelete(ctx, arg)
+	var app *v1.App
+	err := retry.RetryOnConflict(retry.DefaultRetry, func() (err error) {
```

Comment:
does this retry just once or a bunch of times?

Reply:
It's configured to retry with the following configuration:
```
var DefaultRetry wait.Backoff = wait.Backoff{
     Steps:    5,
     Duration: 10 * time.Millisecond,
     Factor:   1.0,
     Jitter:   0.1,
 }
```